kmedvedev/04_CSS/Letter_2.html

```diff
@@ -0,0 +1,63 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title>Письмо 2</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css?family=Bad+Script|Pattaya');
+        body {background-color: #eef3f6; 
+              font-size: 14pt;
+        }
+        header {font-family: serif; 
+                text-align: right;
+        }
+        h1 {font-size: inherit; 
+            font-weight: bold;
+        }
+        adress {font-style: italic;
```

Comment:
address, typo

Reply:
Sorry. I missed this typo because it works fine even with "adress". I think tag <address> has default  cursive font.

Reply:
...tag "address"...